12:29:36.317 [main] INFO  edu.alonso.daw.tema9_world.Init - Inicio del programa...
12:29:45.704 [main] INFO  edu.alonso.daw.tema9_world.repository.CityRepository - Creando el dao para City...
12:29:45.705 [main] INFO  edu.alonso.daw.tema9_world.utils.ConfigLoader - Obteniendo la información de la configuración...
12:29:45.708 [main] INFO  edu.alonso.daw.tema9_world.utils.ConfigLoader - Configuración leída con éxito
12:29:45.709 [main] INFO  edu.alonso.daw.tema9_world.repository.CityRepository - Dao para City creado con éxito
12:29:45.711 [main] INFO  edu.alonso.daw.tema9_world.utils.DBConnection - Creando la conexión con la BBDD...
12:29:45.831 [main] ERROR edu.alonso.daw.tema9_world.utils.DBConnection - Error al crear la conexión con la base de datos.
com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:174) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:64) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:828) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:448) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at java.sql.DriverManager.getConnection(DriverManager.java:681) ~[java.sql:?]
	at java.sql.DriverManager.getConnection(DriverManager.java:229) ~[java.sql:?]
	at edu.alonso.daw.tema9_world.utils.DBConnection.<init>(DBConnection.java:52) [classes/:?]
	at edu.alonso.daw.tema9_world.utils.DBConnection.getInstance(DBConnection.java:31) [classes/:?]
	at edu.alonso.daw.tema9_world.dao.CityDAOMySql.getAll(CityDAOMySql.java:24) [classes/:?]
	at edu.alonso.daw.tema9_world.repository.CityRepository.getAll(CityRepository.java:42) [classes/:?]
	at edu.alonso.daw.tema9_world.service.CityService.listadoCiudades(CityService.java:14) [classes/:?]
	at edu.alonso.daw.tema9_world.Init.main(Init.java:64) [classes/:?]
Caused by: com.mysql.cj.exceptions.CJCommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.exceptions.ExceptionFactory.createCommunicationsException(ExceptionFactory.java:167) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.protocol.a.NativeSocketConnection.connect(NativeSocketConnection.java:89) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.NativeSession.connect(NativeSession.java:120) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:948) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:818) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	... 11 more
Caused by: java.net.ConnectException: Connection refused: connect
	at sun.nio.ch.Net.connect0(Native Method) ~[?:?]
	at sun.nio.ch.Net.connect(Net.java:579) ~[?:?]
	at sun.nio.ch.Net.connect(Net.java:568) ~[?:?]
	at sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:588) ~[?:?]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327) ~[?:?]
	at java.net.Socket.connect(Socket.java:633) ~[?:?]
	at com.mysql.cj.protocol.StandardSocketFactory.connect(StandardSocketFactory.java:153) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.protocol.a.NativeSocketConnection.connect(NativeSocketConnection.java:63) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.NativeSession.connect(NativeSession.java:120) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:948) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:818) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	... 11 more
12:31:39.065 [main] INFO  edu.alonso.daw.tema9_world.Init - Inicio del programa...
12:31:59.465 [main] INFO  edu.alonso.daw.tema9_world.Init - Inicio del programa...
12:32:08.921 [main] INFO  edu.alonso.daw.tema9_world.repository.CityRepository - Creando el dao para City...
12:32:08.922 [main] INFO  edu.alonso.daw.tema9_world.utils.ConfigLoader - Obteniendo la información de la configuración...
12:32:08.925 [main] INFO  edu.alonso.daw.tema9_world.utils.ConfigLoader - Configuración leída con éxito
12:32:08.926 [main] INFO  edu.alonso.daw.tema9_world.repository.CityRepository - Dao para City creado con éxito
12:32:08.927 [main] INFO  edu.alonso.daw.tema9_world.utils.DBConnection - Creando la conexión con la BBDD...
12:32:09.042 [main] ERROR edu.alonso.daw.tema9_world.utils.DBConnection - Error al crear la conexión con la base de datos.
com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:174) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:64) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:828) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:448) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at java.sql.DriverManager.getConnection(DriverManager.java:681) ~[java.sql:?]
	at java.sql.DriverManager.getConnection(DriverManager.java:229) ~[java.sql:?]
	at edu.alonso.daw.tema9_world.utils.DBConnection.<init>(DBConnection.java:52) [classes/:?]
	at edu.alonso.daw.tema9_world.utils.DBConnection.getInstance(DBConnection.java:31) [classes/:?]
	at edu.alonso.daw.tema9_world.dao.CityDAOMySql.hashCity(CityDAOMySql.java:45) [classes/:?]
	at edu.alonso.daw.tema9_world.repository.CityRepository.hashCity(CityRepository.java:46) [classes/:?]
	at edu.alonso.daw.tema9_world.service.CityService.existeCiudad(CityService.java:18) [classes/:?]
	at edu.alonso.daw.tema9_world.Init.main(Init.java:77) [classes/:?]
Caused by: com.mysql.cj.exceptions.CJCommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.exceptions.ExceptionFactory.createCommunicationsException(ExceptionFactory.java:167) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.protocol.a.NativeSocketConnection.connect(NativeSocketConnection.java:89) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.NativeSession.connect(NativeSession.java:120) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:948) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:818) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	... 11 more
Caused by: java.net.ConnectException: Connection refused: connect
	at sun.nio.ch.Net.connect0(Native Method) ~[?:?]
	at sun.nio.ch.Net.connect(Net.java:579) ~[?:?]
	at sun.nio.ch.Net.connect(Net.java:568) ~[?:?]
	at sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:588) ~[?:?]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327) ~[?:?]
	at java.net.Socket.connect(Socket.java:633) ~[?:?]
	at com.mysql.cj.protocol.StandardSocketFactory.connect(StandardSocketFactory.java:153) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.protocol.a.NativeSocketConnection.connect(NativeSocketConnection.java:63) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.NativeSession.connect(NativeSession.java:120) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:948) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:818) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	... 11 more
12:33:31.534 [main] INFO  edu.alonso.daw.tema9_world.Init - Inicio del programa...
12:33:32.701 [main] INFO  edu.alonso.daw.tema9_world.repository.CityRepository - Creando el dao para City...
12:33:32.702 [main] INFO  edu.alonso.daw.tema9_world.utils.ConfigLoader - Obteniendo la información de la configuración...
12:33:32.703 [main] INFO  edu.alonso.daw.tema9_world.utils.ConfigLoader - Configuración leída con éxito
12:33:32.705 [main] INFO  edu.alonso.daw.tema9_world.repository.CityRepository - Dao para City creado con éxito
12:33:32.706 [main] INFO  edu.alonso.daw.tema9_world.utils.DBConnection - Creando la conexión con la BBDD...
12:33:32.928 [main] ERROR edu.alonso.daw.tema9_world.utils.DBConnection - Error al crear la conexión con la base de datos.
java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:828) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:448) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at java.sql.DriverManager.getConnection(DriverManager.java:681) ~[java.sql:?]
	at java.sql.DriverManager.getConnection(DriverManager.java:229) ~[java.sql:?]
	at edu.alonso.daw.tema9_world.utils.DBConnection.<init>(DBConnection.java:52) [classes/:?]
	at edu.alonso.daw.tema9_world.utils.DBConnection.getInstance(DBConnection.java:31) [classes/:?]
	at edu.alonso.daw.tema9_world.dao.CityDAOMySql.getAll(CityDAOMySql.java:24) [classes/:?]
	at edu.alonso.daw.tema9_world.repository.CityRepository.getAll(CityRepository.java:42) [classes/:?]
	at edu.alonso.daw.tema9_world.service.CityService.listadoCiudades(CityService.java:14) [classes/:?]
	at edu.alonso.daw.tema9_world.Init.main(Init.java:64) [classes/:?]
12:46:25.225 [main] INFO  edu.alonso.daw.tema9_world.Init - Inicio del programa...
12:46:29.125 [main] INFO  edu.alonso.daw.tema9_world.repository.CityRepository - Creando el dao para City...
12:46:29.126 [main] INFO  edu.alonso.daw.tema9_world.utils.ConfigLoader - Obteniendo la información de la configuración...
12:46:29.127 [main] INFO  edu.alonso.daw.tema9_world.utils.ConfigLoader - Configuración leída con éxito
12:46:29.128 [main] INFO  edu.alonso.daw.tema9_world.repository.CityRepository - Dao para City creado con éxito
12:46:29.131 [main] INFO  edu.alonso.daw.tema9_world.utils.DBConnection - Creando la conexión con la BBDD...
12:46:29.345 [main] ERROR edu.alonso.daw.tema9_world.utils.DBConnection - Error al crear la conexión con la base de datos.
java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:828) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:448) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198) ~[mysql-connector-j-8.0.31.jar:8.0.31]
	at java.sql.DriverManager.getConnection(DriverManager.java:681) ~[java.sql:?]
	at java.sql.DriverManager.getConnection(DriverManager.java:229) ~[java.sql:?]
	at edu.alonso.daw.tema9_world.utils.DBConnection.<init>(DBConnection.java:52) [classes/:?]
	at edu.alonso.daw.tema9_world.utils.DBConnection.getInstance(DBConnection.java:31) [classes/:?]
	at edu.alonso.daw.tema9_world.dao.CityDAOMySql.getAll(CityDAOMySql.java:24) [classes/:?]
	at edu.alonso.daw.tema9_world.repository.CityRepository.getAll(CityRepository.java:42) [classes/:?]
	at edu.alonso.daw.tema9_world.service.CityService.listadoCiudades(CityService.java:14) [classes/:?]
	at edu.alonso.daw.tema9_world.Init.main(Init.java:64) [classes/:?]
12:48:04.301 [main] INFO  edu.alonso.daw.tema9_world.Init - Inicio del programa...
12:48:06.437 [main] INFO  edu.alonso.daw.tema9_world.repository.CityRepository - Creando el dao para City...
12:48:06.438 [main] INFO  edu.alonso.daw.tema9_world.utils.ConfigLoader - Obteniendo la información de la configuración...
12:48:06.440 [main] DEBUG edu.alonso.daw.tema9_world.utils.ConfigLoader - db_type=mysql
12:48:06.440 [main] DEBUG edu.alonso.daw.tema9_world.utils.ConfigLoader - db_url=jdbc:mysql://127.0.0.1:3306/
12:48:06.440 [main] DEBUG edu.alonso.daw.tema9_world.utils.ConfigLoader - db_name=world
12:48:06.441 [main] DEBUG edu.alonso.daw.tema9_world.utils.ConfigLoader - db_user=root
12:48:06.441 [main] DEBUG edu.alonso.daw.tema9_world.utils.ConfigLoader - db_pass= 
12:48:06.441 [main] INFO  edu.alonso.daw.tema9_world.utils.ConfigLoader - Configuración leída con éxito
12:48:06.442 [main] INFO  edu.alonso.daw.tema9_world.repository.CityRepository - Dao para City creado con éxito
12:48:06.442 [main] INFO  edu.alonso.daw.tema9_world.utils.DBConnection - Creando la conexión con la BBDD...
12:48:06.652 [main] INFO  edu.alonso.daw.tema9_world.utils.DBConnection - Conexión de BBDD creada con éxito!!
12:49:24.131 [main] INFO  edu.alonso.daw.tema9_world.Init - Inicio del programa...
12:49:25.823 [main] INFO  edu.alonso.daw.tema9_world.repository.CityRepository - Creando el dao para City...
12:49:25.824 [main] INFO  edu.alonso.daw.tema9_world.utils.ConfigLoader - Obteniendo la información de la configuración...
12:49:25.826 [main] DEBUG edu.alonso.daw.tema9_world.utils.ConfigLoader - db_type=mysql
12:49:25.826 [main] DEBUG edu.alonso.daw.tema9_world.utils.ConfigLoader - db_url=jdbc:mysql://127.0.0.1:3306/
12:49:25.826 [main] DEBUG edu.alonso.daw.tema9_world.utils.ConfigLoader - db_name=world
12:49:25.826 [main] DEBUG edu.alonso.daw.tema9_world.utils.ConfigLoader - db_user=root
12:49:25.826 [main] DEBUG edu.alonso.daw.tema9_world.utils.ConfigLoader - db_pass= 
12:49:25.826 [main] INFO  edu.alonso.daw.tema9_world.utils.ConfigLoader - Configuración leída con éxito
12:49:25.828 [main] INFO  edu.alonso.daw.tema9_world.repository.CityRepository - Dao para City creado con éxito
12:49:25.829 [main] INFO  edu.alonso.daw.tema9_world.utils.DBConnection - Creando la conexión con la BBDD...
12:49:26.039 [main] INFO  edu.alonso.daw.tema9_world.utils.DBConnection - Conexión de BBDD creada con éxito!!
